modules/dnn/src/protobuf-c/protobuf-c.cpp

```diff
@@ -0,0 +1,3667 @@
+/*
+ * Copyright (c) 2008-2015, Dave Benson and the protobuf-c authors.
```

Comment:
7 years ago.
Do we really need this as a **mandatory** part of the library?
Who would maintain that?

Reply:
yes, we need it